yb-voyager/cmd/common.go

```diff
@@ -87,7 +87,12 @@ func getMappingForTableNameVsTableFileName(dataDirPath string) map[string]string
 		time.Sleep(time.Second * 1)
 	}
 
-	pgRestoreCmd := exec.Command("pg_restore", "-l", dataDirPath)
+	var pgRestorePath string
```

Comment:
Not required. The variable will anyway be defined on the next line.

Reply:
Yeah. Removed.